.grabl/automation.yml

```diff
@@ -63,9 +63,7 @@ build:
         wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
```

Comment:
Update CI scripts: we no longer need username/email/password for authentication when deploying the npm package - we just need a reusable auth token.

The token used depends on which repo we're deploying to, snapshot or release.